src/main/java/uk/gov/hmcts/reform/sscs/model/EntityRoleCode.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@SuppressWarnings({"PMD.AvoidDuplicateLiterals"})
+@Getter
+@RequiredArgsConstructor
+public enum EntityRoleCode {
```

Comment:
This is now in common, please remove this and use the common version

Reply:
ok will delete it